release_notes/47-release-notes.adoc

```diff
@@ -0,0 +1,389 @@
+
+//ROX-14616
+[id="protect-and-run-secured-clusters-on-red-openshift-on-ibm-cloud_{context}"]
+=== Protect and run secured clusters on Red{nbsp}OpenShift on IBM Cloud
+
+In {product-title-short} 4.7, you can now protect {product-roibm} and run secured clusters on {product-roibm}.
+
+//ROX-13542
+[id="use-the-short-lived-oidc-credentials-from-microsoft-azure_{context}"]
+=== Use the short-lived OIDC credentials from Microsoft Azure
+
+With {product-title-short} 4.7, you can now use short-lived OpenID Connect (OIDC) credentials. The Microsoft Azure Identity Provider provides these credentials, and you can use these credentials to access the services in Microsoft Azure, such as the Azure Container Registry and Azure Sentinel.
+
+This feature builds on the {product-title-short} 4.4 release, which introduced the ability to use short-lived OIDC credentials from Amazon Web Services (AWS) and Google Cloud Platform (GCP) cloud providers to authenticate to the services hosted on these platforms.
+
+For more information, see xref:../integration/integrate-with-image-registries.adoc#manual-configuration-image-registry-acr_integrate-with-image-registries[Manually configuring Microsoft Azure Container Registry].
+
+[id="certifications_{context}"]
+== Certifications
+
+{product-title-managed} is certified according to the following global standards for security, compliance, and data protection:
+
+* ISO/IEC 27001:2022
+* ISO/IEC 27017:2015
+* ISO/IEC 27018:2019
+* PCI DSS 4.0
+* SOC 2 Type 2
+* SOC 2 Type 3
+
+[id="notable-technical-changes_{context}"]
+== Notable technical changes
+
+This release contains the following changes:
+
+* Scanner V4 now uses Red{nbsp}Hat VEX files instead of the Common Vulnerabilities and Exposures (CVE) map to provide vulnerability data for non-RPM content in official Red{nbsp}Hat images.
+* You can no longer set the `ROX_NODE_INDEX_CONTAINER_API` environment variable in the Compliance pod. The node scanner never used this variable because the node scanner never connected to the Red{nbsp}Hat Container Catalog.
++
+To enable scanning of Scanner V4 nodes while scanning of Scanner V2 nodes continues in parallel, you must change the `ROX_NODE_INDEX_ENABLED` variable from a BooleanSetting to a feature flag so that it is accessible to the user interface (UI) through the Central API. Central with Scanner V4 automatically prefers Scanner V4 scans, while those with StackRox Scanner V2 continue to work without changes.
+
+* `stackRox.io` Content Delivery Network (CDN) has been moved from CloudFlare to Akamai. When configuring firewall rules, you must use the hostname instead of the IP addresses. If you previously allowed the IP ranges from `stackrox.io`, you must update these rules. 
```

Comment:
```suggestion
* `stackrox.io` Content Delivery Network (CDN) has been moved from CloudFlare to Akamai. When configuring firewall rules, you must use the hostname instead of the IP addresses. If you previously allowed the IP ranges from `stackrox.io`, you must update these rules. 
```